game/plugin/locations/varrock/build.gradle

```diff
@@ -1,10 +1,12 @@
 plugin {
-    name = "varrock_npc_spawns"
+    name = "varrock"
     packageName = "org.apollo.game.plugin.locations"
```

Comment:
Might want to update this to `org.apollo.game.plugin.locations.varrock` so it doesn't conflict with the next shops.plugin.kts added for a different location.